scertecd/scertecd.go

```diff
@@ -676,37 +813,37 @@ func (cu *certUpdateCheck) updateIfNeeded(ctx context.Context, prev *certUpdateR
 	chal, err := cu.getACMEChallenge(ctx)
 	if err != nil {
 		cu.s.addError(errTypeACMEGetChallenge)
-		return nil, fmt.Errorf("getACMEChallenge: %w", err)
+		return fmt.Errorf("getACMEChallenge: %w", err)
 	}
 
 	// Make the DNS record we were told to make to prove we control the DNS.
 	err = cu.s.makeRecord(ctx, cu.Logf, chal.dnsRecordName, chal.dnsRecordValue)
 	if err != nil {
 		cu.s.addError(errTypeMakeRecord)
-		return nil, fmt.Errorf("makeRecord %q: %w", chal.dnsRecordName, err)
+		return fmt.Errorf("makeRecord %q: %w", chal.dnsRecordName, err)
 	}
 	cu.Logf("made DNS record %q", chal.dnsRecordName)
 
 	// Finish the ACME dance.
 	allPEM, err := cu.finishACME(ctx, chal)
 	if err != nil {
 		cu.s.addError(errTypeACMEFinish)
-		return nil, fmt.Errorf("finishACME: %w", err)
+		return fmt.Errorf("finishACME: %w", err)
 	}
 
-	m, err := cu.s.parseCertMeta(allPEM)
+	m, err := cu.s.parseCertMeta(allPEM, false)
```

Comment:
boolean literal arguments are hard to read. Maybe add a `false /* comment */)` here if using a type+const is too onerous?